### PR TITLE
Add manifest for org.kde.isoimagewriter

### PR DESCRIPTION
**Pull request by flyingcakes85**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
   - `--device=all`
     `--filesystem=host`
     `--system-talk-name=org.freedesktop.UDisks2`
     for discovering and writing to USB
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*
   - `0001-Add-content-rating-tag.patch` : https://invent.kde.org/utilities/isoimagewriter/-/merge_requests/15
   - `0001-allow-kauth-to-be-optional-on-linux-with-cmake-DUSE_.patch` : https://invent.kde.org/utilities/isoimagewriter/-/commit/b597702d08b574a86dc4933437c11d48c566ce2f

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by flyingcakes85** [comment]
bot, build org.kde.isoimagewriter

**Comment by flyingcakes85** [comment]
CC @travier

**Comment by travier** [comment]
CC @flathub/kde

File: org.kde.isoimagewriter.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    "sdk": "org.kde.Sdk",
+    "command": "isoimagewriter",
+    "rename-icon": "isoimagewriter",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=all",
+        "--filesystem=host",
```
Comment (barthalion) [inline org.kde.isoimagewriter.json:15]: is host access actually needed?

Comment (hfiguiere) [reply org.kde.isoimagewriter.json:15]: shouldn't `/media` and `/run/media` be enough to access the removable medias to write?

Comment (aleixpol) [reply org.kde.isoimagewriter.json:15]: At the moment, this is using this polkit helper: https://invent.kde.org/utilities/isoimagewriter/-/blob/master/isoimagewriter/imagewriter_helper.cpp

This is used to write the file (given by path, not by fd as the portals like) into the device.
https://invent.kde.org/utilities/isoimagewriter/-/blob/master/isoimagewriter/imagewriter.cpp#L65

I am not sure how any of this is supposed to work on flatpak without changing really...

**Pull request closed without merging**